DQMOffline/EGamma/python/egammaDQMOffline_cff.py

```diff
@@ -31,9 +31,22 @@
 zmumugammaOldAnalysis.analyzerName = cms.string('zmumugammaOldValidation')
 zmumugammaOldAnalysis.phoProducer = cms.InputTag('photons')
 
-
-
+# HGCal customizations
+from Configuration.Eras.Modifier_phase2_hgcal_cff import phase2_hgcal
+stdPhotonAnalysisHGCal = stdPhotonAnalysis.clone()
+stdPhotonAnalysisHGCal.ComponentName = 'stdPhotonAnalyzerHGCalFromMultiCl'
+stdPhotonAnalysisHGCal.analyzerName = 'stdPhotonAnalyzerHGCalFromMultiCl'
+stdPhotonAnalysisHGCal.phoProducer = 'photonsFromMultiCl'
+stdPhotonAnalysisHGCal.isolationStrength = 2
+stdPhotonAnalysisHGCal.etaMin = -3.0
+stdPhotonAnalysisHGCal.etaMax = 3.0
+stdPhotonAnalysisHGCal.maxPhoEta = 3.0
 
 egammaDQMOffline = cms.Sequence(photonAnalysis*stdPhotonAnalysis*zmumugammaOldAnalysis*zmumugammaAnalysis*piZeroAnalysis*electronAnalyzerSequence)
+egammaDQMOfflineHGCal = egammaDQMOffline.copy()
```

Comment:
I cannot do that since if that sequence is not known the next toReplaceWith will produce modules w/o labels.
To do what you suggest I need to move everything done here in the file that does
```
process.load('thisfile')
```

but I preferred to keep the customizations as close as possible to the original source.
What's the penalty of having few obj attached to process?